cal/cal/templates/home_logged_in.html

```diff
@@ -81,59 +80,13 @@ <h4 class="well">Categories - Last Week</h4>
       </ul>
       <nvd3 options="options" data="categories"></nvd3>
     </div>
-    <div ng-controller="TagsCtrl as $ctrl">
+    <div ng-controller="TagsCtrl">
       <h4 class="well">Tags - Cumulative</h4>
-        <table class="table table-hover">
-          <thead>
-            <th>Label</th>
-            <th>Hours</th>
-          </thead>
-          <tbody>
-            <tr ng-repeat="tag in tags">
```

Comment:
Not sure how this happened but you somehow deleted all the functionality that Tiffany had built [here](https://github.com/MaxFangX/calendar-analytics/pull/43). Could you add it back and then reimplement these changes? This looks like the offending commit: [https://github.com/MaxFangX/calendar-analytics/pull/63/commits/91bb9f98fdd447310629f58952950ecb538b205d](https://github.com/MaxFangX/calendar-analytics/pull/63/commits/91bb9f98fdd447310629f58952950ecb538b205d)